### DISCUSSION TITLE
--md-text-link-color has no effect. Was it removed/changed?

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#2157] [Q&A] [answered]
Despite me making a css file with the below content and adding it as extra_css in the mkdocs.yml does it seem like that the `--md-text-link-color` variable has no effect anymore on the primary link color.

Has this value been changed or even removed? And what would be the new way then to alter the link color without also changing other colors (i.e. the primary color)?

CSS file:
```css
:root > * {
    --md-text-link-color: #00BDD6;
}
```

**Comment by squidfunk** [comment, marked as answer]
Yes, it was renamed during refactoring some months ago. It's now `--md-typeset-a-color`. It's probably best to [check the definitions](https://github.com/squidfunk/mkdocs-material/blob/master/src/assets/stylesheets/main/_colors.scss) to learn about all available variables.

**Reply by squidfunk** [reply]
Choosing my comment as an answer to resolve this question.

**Reply by Andre601** [reply]
Sorry. Completely forgot about this post's existance. 😅

**Reply by squidfunk** [reply]
No worries 😊 I just try to keep the discussions as tidy as possible, so users can find answers quickly.

**Reply by Andre601** [reply]
Something I want to ask now is, if there is also a variable for the button color.
I looked at the `_colors.scss` but can't see a var that would hint at it altering the button color. Also, the button color isn't affected by the a-color, so it would at least be a own var.

**Reply by squidfunk** [reply]
The button color is set to the `primary` color + `accent` color on hover. You can just override the `.md-typeset .mb-button` class in extra CSS - no need to define another CSS variable. In fact, the link color is only defined as a CSS variable because it is used in [multiple places](https://github.com/squidfunk/mkdocs-material/search?q=%22--md-typeset-a-color%22).

https://github.com/squidfunk/mkdocs-material/blob/1ba1e1e1d5791a2a1f19021ee078dbd1001efa87/src/assets/stylesheets/main/layout/_button.scss#L31-L58
